docs/graphql-java-comparison.md

```diff
@@ -0,0 +1,146 @@
+---
+id: graphql-java-comparison
+title: GraphQL Java Comparison
+---
+
+[graphql-java](https://graphql-java.com/) is one of the more popular JVM based GraphQL implemenations. GraphQL Kotlin is
+built on top of `grahpql-java` as there are many parts of the GraphQL specification that are trivial and this implemenatation
```

Comment:
Unsure if I would  call the reason why we use it as `trivial pieces of implementation` so maybe we should rephrase that. I'm thinking maybe we should mention rewrite that sentence as

```suggestion
built on top of `grahpql-java` as it can be easily extended with additional functionality and this implementation
```